### ISSUE TITLE
Need to support ONNX operators introduced or changed in opset 12

### ISSUE THREAD

**Issue by winnietsang** [onnx/onnx-tensorflow#561]
Need to support the following operators in opset 12
1. ArgMax
2. ArgMin
3. Einsum
4. MaxPool
5. ReduceMax
6. ReduceMin
7. NegativeLogLikelihoodLoss
8. Dropout
9. BatchNormalization
10. Constant

**Comment by shahirdaya** [comment]
@winnietsang I am working on implementing the Opset 12 changes for Min and Max. Thanks

**Comment by shahirdaya** [comment]
PR [#641](https://github.com/onnx/onnx-tensorflow/pull/641) for Opset 12 and 13 for Max and Min.

**Comment by shahirdaya** [comment]
@winnietsang I am going to work to support Clip in Opset 12.

**Comment by winnietsang** [comment]
@shahirdaya I already open PR #647 to support Clip opset 12 and 13 today

**Comment by shahirdaya** [comment]
@winnietsang awesome. I'll take tackle Dropout.

**Comment by chinhuang007** [comment]
The updated unsupported operator list for opset 12:
1.     Celu
2.     Dropout (PR #671)
3.     Einsum
4.     GatherND
5.     GreaterOrEqual
6.     LessOrEqual
7.     NegativeLogLikelihoodLoss
8.     Pow
9.     SoftmaxCrossEntropyLoss

**Comment by winnietsang** [comment]
The updated unsupported operator list for opset 12:

1. Celu
2. Dropout (Pr #671)
3. Einsum
4. GatherND
5. GreaterOrEqual (PR #744)
6. LessOrEqual (PR #744)
7.  NegativeLogLikelihoodLoss
8. SoftmaxCrossEntropyLoss

**Comment by chinhuang007** [comment]
The updated unsupported operator list for opset 12:

1.     Einsum (PR #773)
2.     NegativeLogLikelihoodLoss
3.     SoftmaxCrossEntropyLoss
